scripts/codestyle/travis_uncrustify.sh

```diff
@@ -0,0 +1,7 @@
+echo $PWD
+if ! $TRAVIS_PULL_REQUEST; then
+	bash ./run_uncrustify.sh -v $(git rev-parse HEAD~1) $(git rev-parse HEAD)
```

Comment:
Modifying script to check only ``src`` would look ugly. Maybe a better option would be to have a file with names of files we want to ignore?